scanners/git-repo-scanner/templates/git-repo-scanner-scan-type.yaml

```diff
@@ -11,7 +11,9 @@ spec:
       {{- if .Values.scannerJob.ttlSecondsAfterFinished }}
       ttlSecondsAfterFinished: {{ .Values.scannerJob.ttlSecondsAfterFinished }}
       {{- end }}
-      backoffLimit: 3
+      {{- if .Values.scannerJob.backoffLimit }}
+      backoffLimit: {{ .Values.scannerJob.backoffLimit }}
+      {{- end }}
```

Comment:
I'd skip the if here and directly reference the backoff limit as we have a non nil default for it.

Reply:
Ok 🤷🏻‍♂️